RecoHI/HiEvtPlaneAlgos/interface/HiEvtPlaneFlatten.h

```diff
@@ -26,7 +36,8 @@
 #include "TMath.h"
 #include <vector>
 
-#define MAXCUT 5000
+#define MAXCUT 10000
+#define MAXCUTOFF 1000
```

Comment:
constants should not be defined with compiler directives; use static const or constexpr;